e2e/testcases/birth/8-validate-declaration-review-page.spec.ts

```diff
@@ -1613,9 +1637,20 @@ test.describe.serial('8. Validate declaration review page', () => {
        */
       await expect(
         page.getByRole('button', {
-          name: `${declaration.child.name.firstNames} ${declaration.child.name.familyName}`
+          name: `${declaration.child.name.firstNames} ${newFamilyNameForChild}`
         })
       ).toBeVisible()
+
+      const element = page
```

Comment:
I see that this is not asserted because of a bug. Is there a ticket for it?

Reply:
Good catch! I'll update it to assert